### PR TITLE
Document 2-arg versions of trim functions

### LINKED ISSUE #20488: [Docs] Document 2-arg versions of trim functions
Presto documentation is missing 2-arg versions of trim, ltrim and rtrim functions. These functions allow to specify characters to trim.

https://prestodb.io/docs/current/functions/string.html#trim

```
presto:di> select trim('test');
 _col0
-------
 test

presto:di> select trim('test', 't');
 _col0
-------
 es

presto:di> select ltrim('test', 't');
 _col0
-------
 est

presto:di> select rtrim('test', 't');
 _col0
-------
 tes

```

CC: @majetideepak @aditi-pandit @tdcmeehan

### PR DESCRIPTION
**Pull request by mbasmanova**
Fixes #20488

```
== NO RELEASE NOTE ==
```

### CODE REVIEW

**Comment by mbasmanova** [comment]
@tdcmeehan Tim, would you help review this PR?

**Review by tdcmeehan — CHANGES_REQUESTED** [review]

File: presto-docs/src/main/sphinx/functions/string.rst [hunk 1]
```diff
@@ -77,6 +77,13 @@ String Functions
 
     Removes leading whitespace from ``string``.
 
+.. function:: ltrim(string, chars) -> varchar
+
+    Removes the longest string containing only characters in ``chars`` from the beginning of the ``string``.
+
+        SELECT ltrim('test', 't'); -- est
+        SELECT ltrim('tetris', 'te'); -- ris
+
```
[the remark this answers is not in the document]
Comment (mbasmanova) [reply presto-docs/src/main/sphinx/functions/string.rst:86]: @tdcmeehan Tim, I can't tell what the changes are. Can you describe these so I can apply?

[the remark this answers is not in the document]
Comment (mbasmanova) [reply presto-docs/src/main/sphinx/functions/string.rst]: @tdcmeehan Made that change. Would you review again?

[the remark this answers is not in the document]
Comment (mbasmanova) [reply presto-docs/src/main/sphinx/functions/string.rst]: Got it. Fixed. Thank you for pointing these out.

**Review by mbasmanova — COMMENTED** [review]
@tdcmeehan Tim, would you take another look?

File: presto-docs/src/main/sphinx/functions/string.rst [hunk 2]
```diff
@@ -77,6 +77,13 @@ String Functions
 
     Removes leading whitespace from ``string``.
 
+.. function:: ltrim(string, chars) -> varchar
+
+    Removes the longest substring containing only characters in ``chars`` from the beginning of the ``string``.
```
Comment (tdcmeehan) [inline presto-docs/src/main/sphinx/functions/string.rst:82]:
```suggestion
    Removes the longest substring containing only characters in ``chars`` from the beginning of the ``string``. ::
```

File: presto-docs/src/main/sphinx/functions/string.rst [hunk 3]
```diff
@@ -103,6 +110,13 @@ String Functions
 
     Removes trailing whitespace from ``string``.
 
+.. function:: rtrim(string, chars) -> varchar
+
+    Removes the longest substring containing only characters in ``chars`` from the end of the ``string``.
```
Comment (tdcmeehan) [inline presto-docs/src/main/sphinx/functions/string.rst:115]:
```suggestion
    Removes the longest substring containing only characters in ``chars`` from the end of the ``string``. ::
```

File: presto-docs/src/main/sphinx/functions/string.rst [hunk 4]
```diff
@@ -193,6 +207,13 @@ String Functions
 
     Removes leading and trailing whitespace from ``string``.
 
+.. function:: trim(string, chars) -> varchar
+
+    Removes the longest substring containing only characters in ``chars`` from the beginning and end of the ``string``.
```
Comment (tdcmeehan) [inline presto-docs/src/main/sphinx/functions/string.rst:212]:
```suggestion
    Removes the longest substring containing only characters in ``chars`` from the beginning and end of the ``string``. ::
```

**Review by tdcmeehan — APPROVED** [review]
LGTM % one additional thing, please make sure the examples are properly formatted (see suggestions).  Feel free to merge once corrected, thank you!

File: presto-docs/src/main/sphinx/functions/string.rst (hunk 2 above)
Comment (mbasmanova) [reply presto-docs/src/main/sphinx/functions/string.rst]: Got it. split_to_map was also broken. Fixed that too.

**Review by steveburnett — APPROVED** [review]
LGTM! I found nothing to suggest that Tim hasn't already commented on.

**Pull request merged**